format_indirect.md

```diff
@@ -0,0 +1,62 @@
+# Indirect Format
+
+The Indirect Format allows for a dependency to be referenced in a smaller format that takes up less space and bandwidth. This is designed primarily for the `src` field in a dependency.
```

Comment:
Remove all references to dependencies. This has nothing to do with dependencies.

Reply:
Will do.